documentation/modules/auxiliary/scanner/mssql/mssql_login.md

```diff
@@ -15,6 +15,41 @@ A docker container can be spun up with the following command to test this module
 
 ## Options
 
+### CreateSession
+
+When using the mssql_login module, the CreateSession option can be used to obtain an interactive
+session within the smb instance. Running the following commands with all other options set:
+
+```
+set CreateSession true
+run
+```
+
+Should give you output containing
+
+```
+[*] MSSQL session 1 opened
+```
+
+Which you can interact with using `sessions -i SESSION_NUMBER` or `sessions -1` to interact with the most recently opened session.
```

Comment:
Thoughts on having the session command output here and then using the appropriate session ID. Think it would add more clarity maybe.